charts/geonode/templates/geonode/geonode-env.yaml

```diff
@@ -21,13 +21,8 @@ data:
   # GeoNode APIs
   API_LOCKDOWN: 'False'
   TASTYPIE_APIKEY: ""
-  # TODO (mwall) set this automatically with helm magics not fully implemented
-  {{ if  ( eq .Release.Revision 1 )}}
-  IS_FIRST_START: 'True'
-  {{ else }}
-  IS_FIRST_START: 'False'
-  {{ end }}
-  FORCE_REINIT: 'False'
+  FORCE_REINIT: {{ .Values.geonode.general.force_reinit | quote }}
```

Comment:
`FORCE_REINIT="true` will run the following invoke scripts at `entrypoint.sh`.

```
    if [ ${FORCE_REINIT} = "true" ]  || [ ${FORCE_REINIT} = "True" ] || [ ! -e "/mnt/volumes/statics/geonode_init.lock" ]; then
        invoke fixtures
        invoke monitoringfixture
        invoke initialized
        invoke updateadmin
    fi
```

They are bascally updating the fixtures. Which makes always sense if the geonode image changed, else it won't do anything. Further it updates the timestamp of the last init process in `initialized` and update the admin password, which I think is mandatory as geonode-k8s users expect the admin password to change when editing the password in the `Values.yaml`

This will slowdown the start process of a new container, but in my opinion its reasonable.